debug.go

```diff
@@ -55,6 +55,15 @@ func describeRequest(op interface{}) (s string) {
 		addComponent("name %q", f.Interface())
 	}
 
+	if f := v.FieldByName("OpContext"); f.IsValid() {
+		meta := f.Interface().(fuseops.OpContext)
+		if meta.Pid == 0 {
+			addComponent("##### NO PID FOR OP ####")
+		} else {
+			addComponent("PID %+v", meta.Pid)
+		}
+	}
+
```

Comment:
@dotslash Similarly to the above, shall we add this to the final PR?

Reply:
Definitely remove `addComponent("##### NO PID FOR OP ####")`

I have a mild preference for `addComponent("PID %+v", meta.Pid)`. Lets see what the owners of jacobsa/fuse say about this.